src/main/java/kr/allcll/backend/session/SessionService.java

```diff
@@ -32,13 +37,31 @@ public CredentialResponse getCredential(String userId) {
     }
 
     public void startSession(String userId) {
+        if (userTaskMap.containsKey(userId) && threadPoolTaskScheduler.isRunning(userTaskMap.get(userId))) {
+            log.info("이미 해당 인증 정보로 세션 갱신 중입니다: {}", userId);
+            return;
+        }
         Credential credential = credentials.findByUserId(userId);
         Runnable resetSessionTask = () -> sessionClient.execute(credential, new EmptyPayload());
-        threadPoolTaskScheduler.scheduleAtFixedRate(resetSessionTask, Duration.ofSeconds(10));
+
+        String taskId = threadPoolTaskScheduler.scheduleAtFixedRate(resetSessionTask, Duration.ofSeconds(10));
```

Comment:
```suggestion
        String taskId = scheduleAtFixedRate(userId, resetSessionTask, Duration.ofSeconds(10));
```
이렇게 하면 `userTaskMap`가 없어도 되지 않을까요?